### DISCUSSION TITLE
Proposal to move Aries VCX to the Open Wallet Foundation

### DISCUSSION THREAD

**Discussion by JamesKEbert** [hyperledger/aries-vcx#1285] [Announcements]
Hi folks, following in the proposal to move ACA-Py to the Open Wallet Foundation (OWF), we, the current maintainers of Aries VCX, are proposing to apply to move Aries VCX to the Open Wallet Foundation. 

We believe this move will result in the project being in the best place to attract consumers, contributors, and improve the overall visibility of the project. We also think co-locating with VCX's related projects (ACA-Py, Credo, and others) would benefit the overall cohesiveness of the community and assist in maintaining and improving project interoperability.

Additionally, efforts presently outside the scope of Aries VCX, such as OID4VC, will be better suited within the scope of the mission of the OWF. This point was well articulated in [Credo/AFJ's proposed move to the OWF](https://github.com/openwallet-foundation/credo-ts/discussions/1586):
>we think that AFJ has a "branding problem." Due to its name and association with other Hyperledger projects, such as Indy, AnonCreds, and Fabric, most people assume that the framework is limited to the Hyperledger ecosystem. This makes it difficult for the framework to be adopted in jurisdictions that rely on non-Aries standards and also limit its appeal to new contributors. We believe that moving the framework to OWF will better communicate its purpose, has the potential to attract a wider audience, increase contributors, and overall adoption of AFJ.

In regards to timing, we are in the active development of a VCX Framework to improve the usability and simplicity of using Aries VCX, and we would like to get the first iteration of this framework completed prior to this move, which we anticipate being soon. 

As the maintainers of VCX, we deeply value the community's input and opinions. At this stage this is only a proposal. If you object to this proposal, please clearly voice your objection here in this thread, detailing why you object.

Additionally, we will be discussing this move September 3rd at the Aries VCX Community Call at 11:00 PM UTC | 4:00 PM PT, and would welcome your participation.

Regards,

The Aries VCX Maintainers

**Comment by gmulhearn** [comment]
https://github.com/openwallet-foundation/project-proposals/pull/54
